service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationRoutes.java

```diff
@@ -211,11 +195,19 @@ private void configureAuditing() {
         .recipientList(exchangeProperty("recipientList"));
 
     // persist audit event
-    from("seda:audit-event")
+    from(ENDPOINT_AUDIT_EVENT)
         .process(
             exchange -> {
               AuditEvent event = exchange.getMessage().getBody(AuditEvent.class);
               auditEventService.logEvent(event);
             });
+
+    from(ENDPOINT_SLACK_EVENT)
+        .filter(exchange -> StringUtils.isNotBlank(masConfig.getSlackExceptionWebhook()))
+        .process(FunctionProcessor.fromFunction(AuditEvent::toString))
+        .to(
+            String.format(
+                "slack:#%s?webhookUrl=%s",
+                masConfig.getSlackExceptionChannel(), masConfig.getSlackExceptionWebhook()));
```

Comment:
@dimitri-amida I believe these lines are causing VRO to fail to start up because `webhookUrl` is set to `null`.

Reply:
But the end 2 end tests do not test MAS routes, only VRO routes

Reply:
Try running VRO locally. I believe the null webhookUrl is causing Camel setup to fail.

Reply:
I don't see an end2end test in the checks. Running [end2end test](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/3585488161) now.

Reply:
Still [failed](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/3585488161/jobs/6033536354#step:7:47). Running VRO locally, the container log shows:
```
Caused by: org.apache.camel.RuntimeCamelException: Missing required endpoint configuration: token or webhookUrl must be defined for Slack producer
	at org.apache.camel.component.slack.SlackEndpoint.createProducer(SlackEndpoint.java:83) ~[camel-slack-3.19.0.jar!/:3.19.0]
	at org.apache.camel.support.DefaultEndpoint.createAsyncProducer(DefaultEndpoint.java:200) ~[camel-support-3.19.0.jar!/:3.19.0]
	at org.apache.camel.processor.SendProcessor.doStart(SendProcessor.java:242) ~[camel-core-processor-3.19.0.jar!/:3.19.0]
...
Caused by: org.apache.camel.RuntimeCamelException: Missing required endpoint configuration: token or webhookUrl must be defined for Slack producer
	at org.apache.camel.component.slack.SlackEndpoint.createProducer(SlackEndpoint.java:83) ~[camel-slack-3.19.0.jar!/:3.19.0]
	at org.apache.camel.support.DefaultEndpoint.createAsyncProducer(DefaultEndpoint.java:200) ~[camel-support-3.19.0.jar!/:3.19.0]
	at org.apache.camel.processor.SendProcessor.doStart(SendProcessor.java:242) ~[camel-core-processor-3.19.0.jar!/:3.19.0]

```

Reply:
#764 fixes the end2end test